src/utils/interaction.py

```diff
@@ -1,12 +1,18 @@
+import os
 from dataclasses import dataclass
 
 import cv2
-from screeninfo import get_monitors
+from screeninfo import Monitor, get_monitors
 
 from src.logger import logger
 from src.utils.image import ImageUtils
 
-monitor_window = get_monitors()[0]
+# If running in a container, make a fake monitor
+monitor_window = (
+    Monitor(0, 0, 1000, 1000, 100, 100, "FakeMonitor", False)
+    if os.environ.get("OMR_CHECKER_CONTAINER")
```

Comment:
I think we shall also disable the call for imshow using this code change -

```
def show(name, origin, pause=1, resize=False, reset_pos=None, config=None):
        if(os.environ.get("OMR_CHECKER_CONTAINER")):
            return

```

Reply:
I required some assistance with these. As of now, I understand that we aim to refrain from displaying image metrics in DOCKER CONTAINER. If I am correct in this understanding, then with my current knowledge, I attempted to address the issue on my end. Please let me know if any clarification is needed.

Reply:
You can test it once by setting show_image_level to 5 in one of the samples, the code should still finish without any errors. If that works fine, above change should suffice.